packages/oscal-react-library/src/components/OSCALCatalogGroup.tsx

```diff
@@ -154,7 +155,7 @@ const CollapsibleListItem: React.FC<CollapsibleListItemProps> = (props) => {
       }}
     >
       <AccordionSummary>{itemText}</AccordionSummary>
-      <AccordionDetails>{children}</AccordionDetails>
+      <MuiAccordionDetails className="AccordionDetails">{children}</MuiAccordionDetails>
```

Comment:
I don't love this... I don't like that three very tightly coupled components get used very differently. Even if we end up just having:

```ts
export const AccordionDetails = (props) => <MuiAccordionDetails {...props} className="AccordionDetails" />;
```

I like that better because it gives consistent usage between them. Not saying I _love_ that idea either but I prefer consistent usage.